packages/xo-web/src/xo-app/backup-ng/new/index.js

```diff
@@ -147,6 +147,7 @@ const getInitialState = () => ({
   compression: undefined,
   crMode: false,
   deltaMode: false,
+  displayAdvancedSettings: false,
```

Comment:
I think `displayAdvancedSettings` should be undefined here.

Reply:
Why ? It works like smartMode no ?

Reply:
It's my bad! I'm working in the refactoring of this code.